src/download.jl

```diff
@@ -0,0 +1,72 @@
+using Printf
+using DataDeps
```

Comment:
It is good practice in Julia to put all `using` statements in the main source file, in this case `Geography.jl`.